client/my-sites/exporter/post-type-options.jsx

```diff
@@ -120,7 +134,7 @@ const PostTypeOptions = React.createClass( {
 					</p>
 				}
 
-				{ shouldShowPlaceholders ? this.renderPlaceholders() : this.renderFields() }
```

Comment:
for what it's worth, I like the ternary the way you had it before. `renderFields()` is somewhat misleading if it does indeed render the placeholders